### PR TITLE
Mobile menu window

### PR DESCRIPTION
**Pull request by baezluise**

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @baezluise, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a pair programming project without commits from other team member(s). Kindly ensure that every member of the team contributes to the project evidenced by their commits.

Also, kindly include a description for your pull request in order to know at a glance the changes implemented.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -10,8 +10,18 @@
     <header>
       <div id="welcome">Welcome</div>
       <nav id="nav">
```
Comment (GzimAsani) [inline index.html:12]: - [ ]  Please give padding to the sides of hamburger button in order to match the design better

Yours:
image

Design:
image

File: index.html [hunk 2]
```diff
@@ -10,8 +10,18 @@
...
       </nav>
+      <div class="modal">
+        <div class="modal-content">
+          <div class="close-button">&times;</div>
+          <ul>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div>
```
Comment (GzimAsani) [inline index.html:23]: - [ ] Kindly check the paddings on the X and the content in order to match better the design


Yours:
image

Design:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @baezluise                   :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with javascript modal, it works great :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers, and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @baezluise, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Optional suggestions
- [ ] **Kindly make sure to add a description to the PR so it reflects the changes in this Branch**
- [ ] Kindly make sure to add a LICENSE to your README file instead so your project has a proper `MIT` License (you can choose any open-source license you want)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by baezluise** [comment]
Mobile menu window added with js events and styled with css

**Pull request merged**